tests/integration_test_net.go

```diff
@@ -189,7 +189,7 @@ func (n *IntegrationTestNet) GetReceipt(txHash common.Hash) (*types.Receipt, err
 	const maxDelay = 100 * time.Millisecond
 	now := time.Now()
 	delay := time.Millisecond
-	for time.Since(now) < 10*time.Second {
+	for time.Since(now) < 100*time.Second {
```

Comment:
what is the motivation for this increase in time ?

Reply:
I have fixed this so many times manually when debugging, and it does not hurt when everything is working fine, so I decided to include it in this PR.

With this increase you have enough time to step through code with the debugger, without risking that you run into a timeout.